### PR TITLE
COMMERCE 5458 3

### PR DESCRIPTION
**Pull request by ivicac**

### CODE REVIEW

**Comment by ivicac** [comment]
ci:test:sf

**Comment by ivicac** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/AvailabilityLabelTag.java [hunk 1]
```diff
@@ -92,6 +111,7 @@ public void setPageContext(PageContext pageContext) {
 
 		_cpContentHelper = ServletContextUtil.getCPContentHelper();
 		servletContext = ServletContextUtil.getServletContext();
+		_productHelper = ServletContextUtil.getProductHelper();
```
Comment (dsitu) [inline modules/apps/commerce/commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/AvailabilityLabelTag.java:114]: Sort

```suggestion
		_cpContentHelper = ServletContextUtil.getCPContentHelper();
		_productHelper = ServletContextUtil.getProductHelper();
		servletContext = ServletContextUtil.getServletContext();
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by ivicac** [comment]
ci:test:sf

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Review by dsitu — APPROVED** [review]

**Comment by ivicac** [comment]
ci:test:sf

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
Resent under https://github.com/liferay-commerce/liferay-portal/pull/694

**Pull request closed without merging**